bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/observability/ObserveUtils.java

```diff
@@ -257,7 +257,7 @@ public static void logMessageToActiveSpan(String logLevel, Supplier<String> logM
         if (!tracingEnabled) {
             return;
         }
-        Environment balEnv = new Environment(Scheduler.getStrand());
+        Environment balEnv = new Environment(Scheduler.getStrand(), null);
```

Comment:
I think it is a bad idea to pass a null to a constructor. We can have two constructors in the `Environment` class to handle this.